fdb-record-layer-lucene/src/main/java/com/apple/foundationdb/record/lucene/directory/FDBDirectory.java

```diff
@@ -175,7 +175,7 @@ private void incrementCallCount(FDBStoreTimer.Counts counter) {
         FDBLuceneFileReference fileReference = this.fileReferenceCache.getIfPresent(name);
         if (fileReference == null) {
             return context.instrument(FDBStoreTimer.Events.LUCENE_GET_FILE_REFERENCE, context.ensureActive().get(metaSubspace.pack(name))
```

Comment:
What thread pool does this run on? The annotation gets added to the method where `asyncToSync` is called, correct? In that case it seems like this method should document what thread pool its future is running on.